conjure_oxide/tests/integration/basic/div/04/input-expected-rule-trace-human.txt

```diff
@@ -0,0 +1,54 @@
+UnsafeDiv(b, c), 
+   ~~> div_to_bubble ([("Bubble", 6000)]) 
+{SafeDiv(b, c) @ (c != 0)} 
+
+--
+
+(a = {SafeDiv(b, c) @ (c != 0)}), 
+   ~~> bubble_up ([("Bubble", 8900)]) 
+{(a = SafeDiv(b, c)) @ And([(c != 0)])} 
+
+--
+
+{(a = SafeDiv(b, c)) @ And([(c != 0)])}, 
+   ~~> expand_bubble ([("Bubble", 8900)]) 
+And([(a = SafeDiv(b, c)), And([(c != 0)])]) 
+
+--
+
+Not(And([(a = SafeDiv(b, c)), And([(c != 0)])])), 
+   ~~> distribute_not_over_and ([("Base", 8400)]) 
+Or([Not((a = SafeDiv(b, c))), Not(And([(c != 0)]))]) 
+
+--
+
+Not((a = SafeDiv(b, c))), 
+   ~~> negated_eq_to_neq ([("Base", 8800)]) 
+(a != SafeDiv(b, c)) 
+
+--
+
+(a != SafeDiv(b, c)), 
+   ~~> flatten_binop ([("Minion", 4400)]) 
+(a != __0) 
```

Comment:
Not for this PR, but something to be captured for future PRs.

This rule is adding a new aux var to the symbol table. In general rules can have global side effects. If so, we should display information about the changes being made to the symbol table here. This is a good example.

@Soph1514

Reply:
Good idea! On my agenda for future pr's

Reply:
I made an issue and assigned it to you, not sure if you noticed?